enchant.lic

```diff
@@ -0,0 +1,181 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#enchant
+=end
+
+custom_require.call(%w[common common-arcana events common-crafting events])
```

Comment:
💡 `events` is in here twice. I don't think that will cause any problems, though.